ansible/inventories/staging/group_vars/v2/vars.yml

```diff
@@ -5,5 +5,6 @@ common_python_version_number: 2.7.16
 
 inventory_ckan_app_version: inventory-bionic
 inventory_ckan_ckanext_saml2_version: bionic-m2crypto
+app_repo_branch: bionic
```

Comment:
Can you prefix any variables with the role name? e.g. `inventory_app_repo_branch`

Reply:
I think we're missing this variable in other inventories, like production but maybe others. Please do a project-wide search/grep for `inventory_ckan_app_version` and add the new variable anywhere you see that.

Reply:
Also, I think after this change, `inventory_ckan_app_version` and `inventory_ckan_ckanext_saml2_version` can probably be removed. Can you check and confirm?